src/cplscheme/CouplingScheme.hpp

```diff
@@ -98,7 +100,54 @@ class CouplingScheme {
    *
    * Does not necessarily advance in time.
    */
-  virtual void advance() = 0;
+  void advance()
+  {
+    firstSynchronization({});
+    firstExchange();
+    secondSynchronization();
+    secondExchange();
+  }
```

Comment:
This is a placeholder, we could now remove, no?

Reply:
Removed